shared/osarch.go

```diff
@@ -51,6 +51,12 @@ var altLinuxArchitectureNames = map[int]string{
 	osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN: "aarch64",
 }
 
+var voidLinuxArchitectureNames = map[int]string{
+	osarch.ARCH_32BIT_INTEL_X86:           "i686",
+	osarch.ARCH_64BIT_INTEL_X86:           "x86_64",
+	osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN: "armv7l",
```

Comment:
Is that the entire set of architectures that void supports?
No arm64 support yet?

Reply:
https://alpha.de.repo.voidlinux.org/live/current/ lists aarch64 and armv6l as well.

Reply:
Ok, we'd probably want to do aarch64 too then, should be added to the map.

Reply:
I've added `aarch64`. Voidlinux also supports `armv6l`, but `ARCH_32BIT_ARMV6_LITTLE_ENDIAN` doesn't exist so I can't add it to the map.